tests/ui/components/results/verticalresultscomponent.js

```diff
@@ -123,7 +127,7 @@ describe('vertical results component', () => {
     });
 
     it('if unset defaults to vertical key', () => {
-      expect(component.getVerticalURL()).toEqual('key.html?query=my-query&otherParam=123');
+      expect(component.getVerticalURL()).toEqual('defaultTabUrl');
```

Comment:
i would think we want to keep the expected the output instead of using this `defaultTabUrl`. is there a reason we changed all of them?